src/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="./styles/styles.css" />
+
+    <title>Bicycle Shop</title>
+  </head>
+
+  <body>
+    <header class="header wrapper">
+      <a href="index.html" class="header__logo">
+        <img src="assets/images/icons/logo.svg" alt="Logo Bicycle Shop" />
+      </a>
+
+      <p class="header__slogan">Paixão pela bike, sede por desafios</p>
+
+      <a href="cart.html" class="header__cart">
+        <img src="assets/images/icons/cart.svg" alt="Carrinho" />
+      </a>
+    </header>
+
+    <main>
+      <form class="wrapper filters">
+        <input class="input" type="search" placeholder="Buscar produto"/>
+
+        <select name="categories" id="dropdown-categories" class="filters-dropdown">
+          <option>Todas as categorias</option>
+          <option>Bicicletas</option>
+          <option>Acessórios</option>
+        </select>
+
+        <select name="filter" id="dropdown-sort" class="filters-dropdown">
+          <option>Menor preço</option>
+          <option>Maior preço</option>
+        </select>
+      </form>
+
+      <section class="wrapper">
+        <ul class="cards grid-col-4">
+          <li class="cards__list">
+            <img class="cards__img" src="assets/images/products/mountain-bike-everest.png" alt="Mountain Bike" />
+
+            <div class="cards__info">
+              <h2 class="cards__product">Mountain Bike Everest</h2>
+
+              <p class="cards__price">R$ 2250,00</p>
+
+              <button class="button">Adicionar ao Carrinho</button>
+            </div>
+          </li>
```

Comment:
Cada `li` seria um bloco `card` e não `cards` no plural, deixei tbm outras sugestões de nomes para alguns elements:

```suggestion
          <li class="card">
            <img class="card__img" src="assets/images/products/mountain-bike-everest.png" alt="Mountain Bike" />

            <div class="card__info">
              <h2 class="card__title">Mountain Bike Everest</h2>

              <p class="cards__text">R$ 2250,00</p>

              <button class="button">Adicionar ao Carrinho</button>
            </div>
          </li>
```